src/Datadog.Trace/TracerConstants.cs

```diff
@@ -9,6 +9,12 @@ internal static class TracerConstants
         /// </summary>
         public const ulong MaxTraceId = 9_223_372_036_854_775_807;
 
+        public static readonly int Major = 1;
```

Comment:
The thought was that this would be useful in any situation where we may want to know specific parts of the version.
We could make an actual Version instance?
`public static readonly Version TracerVersion = new Version("1.21.0.0");`
is that preferable?

Reply:
Are we using these anywhere in this PR? If not, I suggest we revert this change for now and do this in a separate PR. If yes, then is it possible for `Datadog.Core.Tools` to read the version from here and remove `Datadog.Core.Tools/TracerVersion.cs` to avoid duplication?

It does sounds like we could have a single `Version` field that can be used to access each individual part and also the whole string, but again a change like that is better suited for a separate PR.

Reply:
I was using it in the client version for the headers, which are probably not necessary, I was wondering if it would be used in the agent for any correlation in the backend to understand what transport customers are using.

Reply:
I removed the dependent header and reverted this.